app/Controllers/PrestitiController.php

```
@@ -1248,6 +1270,178 @@ public function downloadPdf(Request $request, Response $response, mysqli $db, in
+            }
+            $copyOverlap->close();
+            $update->close();
+            $db->commit();
+        } catch (\Throwable $e) {
+            $db->rollback();
+            SecureLogger::error('Bulk loan extend failed: ' . $e->getMessage());
+            return $response->withHeader('Location', $backUrl . '?error=bulk_extend_failed')->withStatus(302);
+        }
+
+        return $response->withHeader('Location', $backUrl . '?bulk_extended=' . (int) $extended . '&days=' . $days)->withStatus(302);
+    }
```

PHPMD (2.15.0)

[error] Avoid assigning values to variables in if clauses and the like (line '1285', column '13'). (undefined)

(IfStatementAssignment)